### DISCUSSION TITLE
What is the max supported simultaneous remote video streams (maxRemoteVideoStreams)?

### DISCUSSION THREAD

**Discussion by martinesmann** [Azure/communication-ui-library#1755] [Q&A] [answered]
What are the limitations for active remote video streams? 
When using the "VideoGallery" component the maxRemoteVideoStreams can be set to any number, but the component only seems to render up to 6 simultaneous remote streams.
I can't seem to find any documentation on any limit etc. for maxRemoteVideoStreams other than, default is 4. 

Microsoft Teams is able to show up to 25 simultaneous streams I was expecting the same behaviour from the official Microsoft UI library.

**Comment by JamesBurnside** [comment, marked as answer]
For incoming video streams Azure Communication Services currently supports up to 4 streams + 1 screen sharing screen in the browser: https://docs.microsoft.com/en-us/azure/communication-services/concepts/voice-video-calling/calling-sdk-features#calling-sdk-streaming-support. You may observe more than 4 working on some browsers but this is not officially supported.

The VideoGallery UI component in this library has been designed to scale if this number increases in the future.

If your application is in need of a larger number of incoming streams I recommend filing a feature request here detailing your core scenario: https://github.com/Azure/Communication/issues/new/choose. This will route your request to the Team handling this, giving them more visibility to prioritize this.

**Comment by alkwa-msft** [comment]
Closing this discussion due to inactivity. Feel free to re-open if this is still an issue.
